generated/nidaqmx/system/watchdog.py

```diff
@@ -56,10 +56,10 @@ def __init__(self, device_name, task_name='', timeout=10, grpc_options=None):
         """
         self._interpreter = utils._select_interpreter(grpc_options)
 
+        self._handle = None
+        self._close_on_exit = False
 
-        self._handle = lib_importer.task_handle(0)
-
-        self._handle, _ = self._interpreter.create_watchdog_timer_task_ex(device_name, task_name, timeout)
+        self._handle, self._close_on_exit = self._interpreter.create_watchdog_timer_task_ex(device_name, task_name, timeout)
 
         # Saved name is used in self.close() to throw graceful error on
```

Comment:
Please change`__del__` to use `self._saved_name` instead of `self.name`, like it does in `Task`.

Reason: `self.name` calls into the interpreter, which is more likely to raise an exception, especially if the process is shutting down.